### PR TITLE
[598]: hot bage logic

### LINKED ISSUE #598: Change logic of Hot badge
At the moment the Hot badge is assigned to property cards which have been rented atleast once. This has worked when the platform was launched but now as we have more and more rent, the hot badge is on 35% of the properties.

Change the logic for the hot badge to be assigned if:
- the property has been rented >1 (regardless whether the tenant is the same or not)
- the property is currently rented and has an upcoming rent (regardless whether the tenant is the same or not)

### PR DESCRIPTION
**Pull request by mishaadzh**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #598 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: src/modules/land-works/api.ts [hunk 1]
```diff
@@ -1867,6 +1888,7 @@ export async function parseAsset(asset: any): Promise<AssetEntity> {
 
   liteAsset.isAvailable = asset.status === AssetStatus.LISTED;
   liteAsset.availability = getAvailability(liteAsset);
+  liteAsset.isHot = asset.totalRents > 1 && liteAsset.upcomingRents && !liteAsset.availability.isCurrentlyAvailable;
```
Comment (failfmi) [inline src/modules/land-works/api.ts:1891]: why is `!liteAsset.availability.isCurrentlyAvailable` and not `liteAsset.availability.isCurrentlyAvailable`?

Comment (mishaadzh) [reply src/modules/land-works/api.ts:1891]: Becouse in task we have : "when the property is currently rented "

Comment (failfmi) [reply src/modules/land-works/api.ts:1891]: I think that having upcoming rents is enough. You cannot have upcoming rents and be available at the same time.

Comment (mishaadzh) [reply src/modules/land-works/api.ts:1891]: I thought so too. But I decided to do according to the task. Will be fixed

File: src/modules/land-works/api.ts [hunk 2]
```diff
@@ -577,6 +585,7 @@ export type AssetEntity = {
   operator: string;
   status: string;
   rents: RentEntity[];
+  upcomingRents: boolean;
```
Comment (failfmi) [inline src/modules/land-works/api.ts:588]: hasUpcomingRents

File: src/modules/land-works/api.ts [hunk 3]
```diff
@@ -124,6 +128,10 @@ export const USER_SUBSCRIPTION = (paymentToken?: string | null) => gql`
         lastRentEnd
         status
         totalRents
+        rents {
+          start
+          end
+        }
```
Comment (failfmi) [inline src/modules/land-works/api.ts:134]: This has to be added in consumerTo as well

**Pull request merged**